src/main/java/com/eternalcode/lobbyheads/head/HeadBlockService.java

```diff
@@ -0,0 +1,102 @@
+package com.eternalcode.lobbyheads.head;
+
+import com.eternalcode.lobbyheads.configuration.implementation.HeadsConfiguration;
+import com.eternalcode.lobbyheads.notification.NotificationAnnouncer;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.Skull;
+import org.bukkit.entity.Player;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+
+public class HeadBlockService {
+
+    private final HeadsConfiguration config;
+    private final HeadService headService;
+    private final NotificationAnnouncer notificationAnnouncer;
+
+    public HeadBlockService(HeadsConfiguration config, HeadService headService, NotificationAnnouncer notificationAnnouncer) {
+        this.config = config;
+        this.headService = headService;
+        this.notificationAnnouncer = notificationAnnouncer;
+    }
+
+    void removeHead(Player player) {
+        Block block = player.getTargetBlock(null, 5);
+
+        for (HeadInfo headInfo : this.config.heads) {
```

Comment:
Tak się zastanawiam czy nie lepiej było by tu używać mapy Position;HeadInfo żeby przy większej ilości takich główek działało to szybciej.

Reply:
Zrobię to.